scaladoc/src/dotty/tools/scaladoc/renderers/HtmlRenderer.scala

```diff
@@ -58,19 +58,18 @@ class HtmlRenderer(rootPackage: Member, val members: Map[DRI, Member])(using ctx
 
   val hiddenPages: Seq[Page] =
     staticSite match
-      case None =>
-        Seq(navigablePage.copy( // Add index page that is a copy of api/index.html
-          link = navigablePage.link.copy(dri = docsRootDRI),
-          children = Nil
-        ))
+      case None => Nil
```

Comment:
Don’t we still need to copy `api/index.html` in case of `-Ylegacy-api-layout`?

Reply:
Good catch!